src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -172,4 +178,59 @@ private function getAvailableFormatOptions(): array
     {
         return (new DescriptorHelper())->getFormats();
     }
+
+    private function sortBy(RouteCollection $routeCollection, string $column): RouteCollection
+    {
+        $routesArray = $routeCollection->all();
+        $sortedRoutes = new RouteCollection();
+
+        if ('name' === $column) {
+            \ksort($routesArray);
+            foreach ($routesArray as $name => $route) {
+                $sortedRoutes->add($name, $route);
+                $sortedRoutes->addAlias($route->getDefault('_controller'), $name);
+            }
+
+            return $sortedRoutes;
+        }
+
+        $helperArray = [];
+
+        if ('method' === $column) {
+            foreach ($routesArray as $name => $route) {
+                $methods = empty($route->getMethods()) ? 'ANY' : \implode('|', $route->getMethods());
```

Comment:
Since the logic for formatting a list of methods is in the descriptors, it would make sense to move the sorting method into `Symfony\Bundle\FrameworkBundle\Console\Descriptor\Descriptor`. The method `describeRouteCollection` has an `$options` array that could be used to pass the desired sort key.
That would make unit testing a lot easier.